CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 Changelog is kept with respect to version 0.11 of Entropies.jl. From version v2.0 onwards, this package has been renamed to ComplexityMeasures.jl.
 
+## 3.8
+
+- Fixed a bug in the `Probabilities` constructor. It is now again possible to construct `Probabilities`
+  when outcome vectors have different element types.
+ 
```

Comment:
Bug fixes do not increment minor number, they increment patch number, see https://semver.org/

Reply:
I don't think we should state this in the changelog either. In my opinion changelog is really about the things a user would care to read about, and bugfixes is not it. they inflate the changelog unecessarily.